docs/2023/CORD Network/IntegrationWithSunbirdRC/updates/2023-08-11.md

```diff
@@ -5,13 +5,41 @@ author: Venu Choudhary
 
 ## Milestones
 
-- [ ] Give the description about Milestone 1
-- [ ] Give the description about Milestone 2
-- [ ] Give the description about Milestone 3
-- [ ] Give the description about Milestone 4
+- [x] Using SunbirdRC's APIs
 
-## Screenshots / Videos
+### Milestone 1 : Using Sunbird RC's APIs
 
-## Contributions
+Sunbird RC provides us with a set of autogenerated APIs which is one of the USP of this plaform. Now in order to use these APIs, most of the times we need to be authenticated, and in the previous week we have already made the keycloak setup,the authentication provider for Sunbird.
+
+During this week, I worked on the schema creation APIs. [Links](https://docs.sunbirdrc.dev/api-reference/schema/create-schema)
+
+Now, when we hit this endpoint, we need to make sure we are authorized. We will need to authenticate as the admin user we created previous week, along with the access token.
+
+```bash
+    http://keycloak:8080/auth/realms/sunbird-rc/protocol/openid-connect/token
```

Comment:
This is not working. May you mention what is this for.

Reply:
This is basically for authenticating users. When you hit this endpoint, we get an access token.